deploy/monai-deploy-express/sample-workflows/liver-tumor.json

```diff
@@ -0,0 +1,84 @@
+{
+	"name": "ai-liver-seg",
+	"version": "1.0.0",
+	"description": "AI Liver Tumor 2.0",
+	"informatics_gateway": {
+		"ae_title": "MONAI-DEPLOY",
+		"data_origins": [
+			"ORTHANC"
+		],
+		"export_destinations": [
+			"ORTHANC"
+		]
+	},
+	"tasks": [
+		{
+			"id": "router",
+			"description": "Ensure series description contains liver",
+			"type": "router",
+			"task_destinations": [
+				{
+					"name": "liver",
+					"conditions": ["{{ context.dicom.series.any('0008','103E')}} == 'CT series for liver tumor from nii 014'"]
+				}
+			]
+		},
+		{
+			"id": "liver",
+			"description": "Execute Liver Segmentation MAP",
+			"type": "docker",
+			"args": {
+				"container_image": "ghcr.io/mmelqin/monai_ai_livertumor_seg_app_stl-x64-workstation-dgpu-linux-amd64:2.0",
+				"server_url": "unix:///var/run/docker.sock",
+				"user": "1000:1000",
+				"entrypoint": "/bin/bash,-c",
+				"command": "python3 -u /opt/holoscan/app/",
```

Comment:
This can safely be removed along with the entrypoint (together), as we made sure that the MAP container will execute the application by default, handled internally in the MAP.

Explicitly specifying the command is only needed for MAP built with earlier version of Packager, namely v0.5.1 and below, because the default command/entrypoint had some issues that did not get addressed by the contributor. In v0.6, we have the new internal handler, and do not need to carry on with the workaround.

Reply:
Unfortunately, it is currently required by the Docker plug-in.  We can do it in another PR.